src/Directory.Build.targets

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <Import Project="../Directory.Build.targets" />
+
+  <ItemGroup>
+    <BuiltBinary Include="$(TargetPath)" />
+  </ItemGroup>
+
+  <!-- Target used to consolidate all PDBs into a single location -->
```

Comment:
Can we expand on this comment further?

Reply:
I don't know what you would like to see. All product PDBs go into a specific location. The reasoning seems to be to help packaging. I don't really know what else to say other than the present comment. I don't want the litter the code with policy only functionality.

Reply:
We are only using arcade to build assets, not rely on arcades output paths at the moment. That is a second pass.